api/api/openapi.html

```diff
@@ -0,0 +1,312 @@
+      </li>
+      {% endfor %}
+    </ul>
+  </div>
+  {% endif %}
+{% else %}
+  <span class="has-text-weight-semibold">
+    {{ property }}
+    {% if schema.type %}<span class="tag">{{ schema.type }}</span>{% endif %}
+  </span>
+  {% if "description" in schema %}
+  <p class="has-text-weight-light">{{ render_markdown(schema.description) }}</p>
+  {% endif %}
+{% endif %}
+{% endmacro %}
+
+{% macro render_schema(schema, render_markdown) %}
+<h2 class="title is-2">{{ schema.info.title }} v{{ schema.info.version }}</h2>
+<p>{{ render_markdown(schema.info.description) }}</p>
+
+<dl class="is-horizontal">
+  <dt>Terms of Service</dt>
+  <dd>{{ schema.info.termsOfService }}</dd>
+
+  <dt>Contact</dt>
+  <dd>
+    <a href="mailto:{{ schema.info.contact.email }}">{{ schema.info.contact.name }}</a>
+  </dd>
+
+  <dt>License</dt>
+  <dd>{{ schema.info.license.name }}</dd>
+</dl>
+
+<h3 class="title is-3">Operations</h3>
+<ul>
+  {% for path, path_schema in schema.paths.items() %}
+    {% for method, method_schema in path_schema.items() %}
+    <li>
+      <a href="#{{ method_schema.operationId }}">
+        <span class="is-family-monospace is-uppercase">{{ method }}</span>
+        <span class="is-family-monospace">{{ path }}</span>
+        <span class="">({{ method_schema.operationId }})</span>
+      </a>
+      {% if method_schema.summary %}
+      - <span>{{ method_schema.summary }}</span>
+      {% endif %}
+      {% if method_schema.responses %}
+      <ul>
+        {% for response_key, response_schema in method_schema.responses.items() %}
+        <li>
+          <a href="#{{ method_schema.operationId }}_responses_{{ response_key }}">
+            {{ response_key }}
+          </a>
+        </li>
+        {% endfor %}
+      </ul>
+      {% endif %}
+    </li>
+    {% endfor %}
+  {% endfor %}
+</ul>
+
+{% for path, path_schema in schema.paths.items() %}
+  {% for method, method_schema in path_schema.items() %}
+  <hr />
+  <div class="section method-schema-section">
+
+    <a href="#{{ method_schema.operationId }}">
+      <h3 class="title is-3" id="{{ method_schema.operationId }}">
+        <span class="is-family-monospace is-uppercase">{{ method }}</span>
+        <span class="is-family-monospace">{{ path }}</span>
+        <span class="">({{ method_schema.operationId }})</span>
+      </h3>
+    </a>
+
+    {% if method_schema.summary %}
+    <p>{{ render_markdown(method_schema.summary) }}</p>
+    {% endif %}
+
+    {% if method_schema.description %}
+    <p>{{ render_markdown(method_schema.description) }}</p>
+    {% endif %}
+
+    {% if method_schema.requestBody %}
+    <h4 class="title is-4">Request body</h4>
+
+    {% for content_type, content_schema in method_schema.requestBody.content.items() %}
+      <a href="#{{ method_schema.operationId }}_requestBody_{{ response_key }}_{{ content_type }}">
+        <h6 class="title is-6" id="{{ method_schema.operationId }}_requestBody_{{ response_key }}_{{ content_type }}">
```

Comment:
A11y rules: this element should use `<h5>` to be correctly cascading headings, unless I'm not reading the inheritance correctly. If it doesn't display nicely, use text styles for override or get the designer to fix it. :)